src/main/java/com/github/checkstyle/regression/Main.java

```diff
@@ -34,6 +68,91 @@ private Main() {
      * @throws Exception execute failure
      */
     public static void main(String[] args) throws Exception {
-        // empty for now
+        final Options options = createOptions();
+        final CommandLineParser parser = new DefaultParser();
+        final HelpFormatter formatter = new HelpFormatter();
+        final CommandLine cmd;
+
+        try {
+            cmd = parser.parse(options, args);
+            final String repoPath = cmd.getOptionValue(OPT_REPO);
```

Comment:
All paths and other fields need to be validated if they are entered.
Paths must exist and be a directory, etc..
etc..